### ISSUE TITLE
cpp/recipe/random_reproducibility

### ISSUE THREAD

**Issue by tocic** [hackingcpp/comments#22]
# (Non-)Reproducible Random Number Sequences | hacking C++

How to obtain reproducible / unpredictable random number sequences?

https://hackingcpp.com/cpp/recipe/random_reproducibility.html

**Comment by tocic** [comment]
Hello, thanks for the great resource.

Please consider changing the `Dice` constructor in the last example from

```cpp
explicit
Dice(seed_type seed = 0);
```

to

```cpp
Dice();
explicit Dice(seed_type seed);
```

to unblock copy-list-initialization. See [P0935](https://wg21.link/p0935) for details.

**Comment by hackingcpp** [comment]
@tocic good point! I totally forgot about this, despite having been bitten by this issues a couple of times in the past.
I updated the example accordingly.
